### PR TITLE
Implement t.drag and t.dragToElement (part of #240)

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

**Comment by inikulin** [comment]
lgtm

File: test/functional/fixtures/api/es-next/drag/test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    describe('t.dragToElement', function () {
+        it('Should drag an element to another element', function () {
+            return runTests('./testcafe-fixtures/drag-test.js', 'Drag to element', { shouldFail: true, only: 'chrome' })
+                .catch(function (errs) {
+                    expect(errs[0]).to.contain('Drag to element completed successfully');
+                    expect(errs[0]).to.contain('> 14 |    await t.dragToElement(\'#draggable-div-2\', \'#destination-div\'');
+                });
+        });
+
+        it("Should fail if a dragged element doesn't exist", function () {
```
Comment (AlexanderMoskovkin) [inline test/functional/fixtures/api/es-next/drag/test.js]: This test and tests below check just client logic, but it's tested in the raw-api tests.
In other es-next api actions tests we check:
1) base action running
2) option validation on the server (see [here](https://github.com/DevExpress/testcafe/blob/master/test/functional/fixtures/api/es-next/click/test.js)

So we should leave the first two steps and replace others with test for arguments/options validation

**Comment by AndreyBelym** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm
/cc @VasilyStrelyaev

**Comment by AndreyBelym** [comment]
It needs to be rebased now.

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**